resources/views/deletefaq.blade.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Illuminate\Support\Facades\DB
+?>
+
+<x-layout>
+    <x-slot:title>
+        Delete FAQ Post {{ $post }}
```

Comment:
Bad variable name here. This page is not about 'posts' but about 'faqs'. The variable holds the id of the faq that must be deleted.